core/src/cluster_nodes.rs

```diff
@@ -233,8 +235,10 @@ impl ClusterNodes<RetransmitStage> {
                 0
             } else if self_index <= fanout {
                 1
-            } else {
+            } else if self_index <= fanout.saturating_add(1).saturating_mul(fanout) {
                 2
+            } else {
+                3 // If changed, update MAX_NUM_TURBINE_HOPS.
```

Comment:
i know we _shouldn't_ be able to make it down there, but future bugs happen.  should we give the no drop `root_distance` calculation below support for the extra level?

Reply:
done